go-app-ussd_chw.js

```diff
@@ -379,23 +379,66 @@ go.app = function() {
             var registration_info = {
                 "reg_type": "momconnect_prebirth",
                 "registrant_id": self.im.user.answers.registrant.id,
+                "registration_type" : self.im.user.answers.registration_type,
```

Comment:
This isn't how we set the registration type on SEED. It's set on the `reg_type` field as either `momconnect_prebirth` or `whatsapp_momconnect_prebirth`

Reply:
Is there a way to differentiate a whatsapp subscription to a non-whatsapp one?

Reply:
Yes, the `reg_type` field determines what kind of registration it is.

For the `momconnect_prebirth`, it's an SMS registration, for `whatsapp_prebirth`, is a whatsapp registration